Foundation/NSXMLParser.swift

```diff
@@ -254,10 +249,8 @@ internal func _NSXMLParserStartElementNs(_ ctx: _CFXMLInterface, localname: Unsa
             var namespaceNameString: String?
             var asAttrNamespaceNameString: String?
             if let ns = namespaces[idx] {
-                if reportNamespaces {
-                    namespaceNameString = UTF8STRING(ns)
-                }
-                asAttrNamespaceNameString = _colonSeparatedStringFromPrefixAndSuffix("xmlns", 5, ns, UInt(strlen(unsafeBitCast(ns, to: UnsafePointer<Int8>.self))))
+                namespaceNameString = UTF8STRING(ns)
```

Comment:
No worries, thanks for looking it over, it made me double check the changes. Could you run a swift-ci test on this PR please.